packages/pds/tests/sync/subscribe-repos.test.ts

```diff
@@ -161,6 +161,7 @@ describe('repo subscribe repos', () => {
     const isDone = async (evt: any) => {
       if (evt === undefined) return false
       if (evt instanceof ErrorFrame) return true
+      await ctx.sequencerLeader.isCaughtUp()
```

Comment:
Can this line be removed?

Reply:
That makes sense to me in the other places this is checked, but I still feel like I may be missing something here.  Is it possible it should be something similar to this?
```js
const caughtUp = await ctx.sequencerLeader.isCaughtUp()
if (!caughtUp) return false
```

Reply:
oh that's absolutely right ^^ good catch